test/lib/pipelineFactory.test.js

```diff
@@ -85,8 +85,7 @@ describe('Pipeline Factory', () => {
                 data: {
                     admins,
                     createTime: nowTime,
-                    scmUrl,
-                    configUrl: scmUrl
+                    scmUrl
```

Comment:
This was fixed from St.John's previous PR. I'm curious why it didn't show merge conflicts. 
But I rebased and it automatically fixed itself. :)